triton-flash-attention.py

```diff
@@ -585,18 +606,25 @@ def run_flash_attention_once(q, k, v, o, cu_seqlens_q, cu_seqlens_k, max_q, max_
 
     print("\n=== Measuring Triton Flash Attention Kernel Startup Time ===")
 
-    # First run (Cold Start)
+    # First run (Cold Start), JIT warm-up
+    run_flash_attention_once(*inputs)
+    
+    # Clear Triton cache
+    clear_triton_cache()
```

Comment:
But if I run this in the container with the cache mounted it will clear the cache which is what we don't want, right?

Reply:
I added this in here https://github.com/maryamtahhan/q1-triton-demo/commit/4f62a54d9a69e36b7225eb907298b20719a95100

Reply:
thinking about this - does this not effectively clear the JIT warmup and includes it again in the startup time after the cache clears?